test/test_bundled_images.py

```diff
@@ -2,10 +2,13 @@
 import torch
 import torch.utils.bundled_inputs
 import io
-import cv2
-from torch.testing._internal.common_utils import TestCase
+from torch.testing._internal.common_utils import TestCase, IS_FBCODE
+
+if not IS_FBCODE:
```

Comment:
Better add a comment here. What happens if this test file is run using unittest?

Reply:
From OSS, it doesn't work. But that's true today anyway.

Reply:
If this adds a comment explaining the pytest skip maybe a comment that this test won't work in OSS would be friendly, too? I guess there's a TODO to make this file internal-only or skip it when run with unittest, too.

Reply:
Dumping this part of the patch, this test needs to go.